pyproject.toml

```diff
@@ -0,0 +1,41 @@
+[build-system]
+requires = ["setuptools>=64.0.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "stylia"
+dynamic = ["version"]
+authors = [
+    {name = "Miquel Duran-Frigola, Ersilia Open Source Initiative", email = "miquel@ersilia.io"},
+]
+description = "Helper functions used by Ersilia to produce presentable plots with MatPlotLib"
+readme = "README.md"
+license = {text = "MIT"}

```

Comment:
Thanks for asking about this! Yes, this could indeed just become license = "MIT". I chose to do it with the table format license = {text = "MIT"} mainly because it follows the newer PEP 621 standard and gives us more flexibility if we need to add more license details later. But you're absolutely right that the simpler version would work just fine too.
I went with the more explicit format mostly to stay consistent with modern Python packaging practices, but if you think we should simplify it, I'm open to that. It's really just a matter of preference - both ways will get the job done. Let me know what you think!

Reply:
Makes sense, thank you!